<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaderVertical</class>
 <widget class="QWidget" name="FaderVertical">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>120</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> dB</string>
   </property>
   <property name="minimum">
    <double>-200.000000000000000</double>
   </property>
   <property name="maximum">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_quiet">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>185</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">quiet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_loud">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string notr="true">loud</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>51</width>
     <height>171</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="pageStep">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
   <property name="tickInterval">
    <number>10000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
